sdks/java/io/gcp/bigtable/src/test/java/org/apache/beam/sdk/io/bigtable/BigtableIOTest.java

```diff
@@ -57,12 +53,10 @@
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.protobuf.ByteString;
 import com.google.protobuf.Empty;
+import org.apache.beam.sdk.io.bigtable.BigtableIO.BigtableSource;
 
 import org.hamcrest.Matchers;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
We don't have any `*` imports, I believe. Let's try to follow the same pattern to avoid this.